bower.json

```diff
@@ -3,5 +3,10 @@
   "main": [
     "src/js.cookie.js"
   ],
-  "ignore": []
+  "ignore": [
+    "travis.sh",
+    "test",
```

Comment:
@timse As from [this comment](https://github.com/js-cookie/js-cookie/pull/83#issuecomment-143968146), doesn't it makes sense removing everything then, including the dot files? Like `.gitignore`, `.jshintignore`, `.jshintrc`, `.travis.yml` and `.tm_properties`?

It seems that the only things that matters are `README.md`, `MIT-LICENSE.txt`, `CONTRIBUTING.md` (is it needed?), and `src/js.cookie.js`, right?